index.bs

```diff
@@ -4639,6 +4639,121 @@ settings:</p>
  Objects</a> not part of a region</li>
 </ul>
 
+<p>The 'color' property on <a>WebVTT Internal Node Objects</a> that have one or more <a lt="WebVTT
+Node Object's applicable classes">applicable classes</a> matching those in the table below, must be
+set, as <a spec=html>presentational hints</a>, to the value in the second cell of the row of the
+table below whose first cell is the value of the corresponding <a lt="WebVTT Node Object's
+applicable classes">applicable class</a>:</p>
+
+<table class="complex data">
+ <thead>
+  <tr>
+   <th><a lt="WebVTT Node Object's applicable classes">applicable class</a></th>
+   <th>'color' value</th>
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td><code>white</code></td>
+   <td>''rgba(255,255,255,1)''</td>
+  </tr>
+  <tr>
+   <td><code>green</code></td>
+   <td>''rgba(0,255,0,1)''</td>
+  </tr>
+  <tr>
+   <td><code>cyan</code></td>
+   <td>''rgba(0,255,255,1)''</td>
+  </tr>
+  <tr>
+   <td><code>red</code></td>
+   <td>''rgba(255,0,0,1)''</td>
+  </tr>
+  <tr>
+   <td><code>yellow</code></td>
+   <td>''rgba(255,255,0,1)''</td>
+  </tr>
+  <tr>
+   <td><code>magenta</code></td>
+   <td>''rgba(255,0,255,1)''</td>
+  </tr>
+  <tr>
+   <td><code>blue</code></td>
+   <td>''rgba(0,0,255,1)''</td>
+  </tr>
+  <tr>
+   <td><code>black</code></td>
+   <td>''rgba(0,0,0,1)''</td>
+  </tr>
+ </tbody>
+</table>
+
+<p class=note>The color for the class <code>green</code> is <em>not</em> the same as the CSS named
```

Comment:
This issue has come up before and the TTWG resolved to retain the CSS definition of `"green"` in TTML but note that other standards' "green" is in fact equivalent to `"lime"` in CSS. This was in the context of TTML, but it makes sense to adopt the same approach here I think. I think it's confusing to have named colors that are other than those in CSS, although for TTML we did introduce some synonyms like "cyan" which is the same as "aqua".

Reply:
Curious to hear from others, since "green" in captioning has such an ingrained use. WebVTT cues are built as a captioning spec and CSS is just used for rendering so we could keep the captioning history. OTOH we could also use this to transform captioning practice to the more widely used CSS color names.